llvm/test/CodeGen/AArch64/srem-vec-crash.ll

```diff
@@ -0,0 +1,14 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+;RUN: llc -mtriple=aarch64-unknown-unknown < %s | FileCheck %s
+
+define i32 @f(i1 %0) {
```

Comment:
Use issue number in function / test name?

Reply:
Done